### PR TITLE
feat(create-post):created add image functionality in create-post

### PR DESCRIPTION
**Pull request by Aditya3034**
### Functionality:
Created add image functionality to create-post

### Solution:
Changes in create-post , posts , upload-file service , social-page and librarian components.

### Risk level:
- [X] high 
- [ ] medium
- [ ] low

### How to test:
Steps to test:-
1. open emulator
2. click on create post in Techtag.
3. click on photo and select the image.
4. click on post to post the image in social page.
5. To check the basepath go to firebase storage.

### CODE REVIEW

File: functions/model/librarian/tark/uploadPostImages.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+       FileUrl: fileUrl,
+       LastModified: lastModified,
+       Size: size,
+       Date: date,
+       Time: time,
+       OrgFileDocumentName: orgFileDocumentName,
+       BasePath: basePath,
+       FileStatus: "OK",
+     };
+     setPostImages(updateUserFileJson, PostId, imageFileName);
```
Comment (viv9k) [inline functions/model/librarian/tark/uploadPostImages.js:57]: postId should be camel case

**Review by viv9k — CHANGES_REQUESTED** [review]
Please do the required fix

File: src/app/services/fileUploadService/file-upload.service.ts [hunk 2]
```diff
@@ -11,7 +11,7 @@
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. 
 * See the MIT License for more details. 
 ***********************************************************/
-import { Injectable } from '@angular/core';
+import { Injectable,Input } from '@angular/core';
```
Comment (viv9k) [inline src/app/services/fileUploadService/file-upload.service.ts:14]: what is the use of Input?

File: src/app/body/view-organization-details/teams-card/teams-card.component.ts [hunk 3]
```diff
@@ -59,26 +59,26 @@ export class TeamsCardComponent implements OnInit {
...
+      next: (data) => {
+        this.team.TeamStatus = -1;
+      },
+      error: (error) => {
+        console.error("Error", error);
+        this.errorHandlerService.showError = true;
+        this.errorHandlerService.getErrorCode(this.componentName, "InternalError","Api");
+      },
+      complete: () => console.info('Successful ')
+  });
```
Comment (viv9k) [inline src/app/body/view-organization-details/teams-card/teams-card.component.ts:80]: Why to change upload team mechanism?

**Review by viv9k — CHANGES_REQUESTED** [review]
Please check properly and update

**Review by viv9k — APPROVED** [review]
Approved

**Review by Pendyala-thanvi — CHANGES_REQUESTED** [review]
Resolve merge conflicts

**Review by Pendyala-thanvi — APPROVED** [review]

**Pull request merged**